src/chrome/content/rules/Whisky.de.xml

```diff
@@ -0,0 +1,23 @@
+<!--
+	Invalid certificate:
+		- download.whisky.de
+		- webdisk.news.whisky.de
+
+	Login required:
+		- test.whisky.de
+
+	Unable to connect:
+		- pdf.whisky.de
+		- remote.whisky.de
+-->
+<ruleset name="Whisky.de">
+	<target host="whisky.de" />
+	<target host="www.whisky.de" />
+	<target host="news.whisky.de" />
+	<target host="proxy.whisky.de"/>
+	<target host="shophe.whisky.de"/>
+	<target host="typo3.whisky.de"/>
+	<target host="vanilla.whisky.de"/>
```

Comment:
This host uses a wildcard DNS record and cert.

Reply:
Done.